### PR TITLE
MM-13422: Fix manage teams modal usage in system console.

### PR DESCRIPTION
**Pull request by grundleborg**
#### Summary
Fix manage teams modal usage in system console to not pass the team ID where a User ID is expected.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-13422

### CODE REVIEW

File: components/admin_console/manage_teams_modal/manage_teams_modal.jsx [hunk 1]
```diff
@@ -101,6 +101,10 @@ export default class ManageTeamsModal extends React.Component {
         }
     }
 
+    handleMemberChange = async () => {
```
Comment (sudheerDev) [inline components/admin_console/manage_teams_modal/manage_teams_modal.jsx]: Do we need async here?

Comment (grundleborg) [reply components/admin_console/manage_teams_modal/manage_teams_modal.jsx]: I don't actually know! In that method I'm calling directly another `async` method, so I assumed this one had to be `async` too, but perhaps that is an incorrect assumption?

Comment (sudheerDev) [reply components/admin_console/manage_teams_modal/manage_teams_modal.jsx]: @grundleborg I checked and you don't need async here. so, we can safely remove it
Here you don't need to await for any async actions so we can just directly call `getTeamMembers `.

Comment (grundleborg) [reply components/admin_console/manage_teams_modal/manage_teams_modal.jsx]: Done.

Comment (sudheerDev) [reply components/admin_console/manage_teams_modal/manage_teams_modal.jsx]: just one more thing while we are at it. Incase if you needed async you would have to have a `await` inside the function or return the async function inside to await any function that calls this async function.

Comment (grundleborg) [reply components/admin_console/manage_teams_modal/manage_teams_modal.jsx]: Thanks dude. Really appreciate you sharing your javascript knowledge, as I'm not very familiar with these bits of the language :)

**Pull request merged**